src/utils.ts

```diff
@@ -44,7 +44,7 @@ export function objectify(arg: any): any {
  * E.g. "foo-bar" -> "fooBar"
  */
 export function camelize(stringWithHyphens: string): string {
-    return stringWithHyphens.replace(/-(.)/g, (_, nextLetter) => nextLetter.toUpperCase());
+    return stringWithHyphens.replace(/-(.)/g, (_, nextLetter) => (nextLetter as string).toUpperCase());
```

Comment:
um, I don't think that's quite right @andy-hanson. type assertions (`as string`) are generally less safe than typedefs, because the assignability checks for the latter are stricter